src/styles/blocks/contact.scss

```diff
@@ -0,0 +1,148 @@
+@mixin footer-padding {
+  padding: 0 20px;
+
+  @media (min-width: $media-tablet) {
+    padding: 0 72px;
+  }
+
+  @media (min-width: $media-desktop) {
+    padding: 0 120px;
+  }
+}
```

Comment:
Could you store the mixins in a separate file? Maybe, it will be more convenient for you.